data/map.txt

```diff
@@ -2549,6 +2549,12 @@ system Algenib
 			sprite planet/dust7-b
 			distance 147
 			period 13.5253
+		object
+			sprite "planet/defense asteroid"
+			distance 147
+			period 13.5253
+			offset 165
+			defense "Defense Asteroid"
```

Comment:
So `defense` references a ship, and that ship has its own sprite, but the object also has a defined sprite. So if no object sprite is defined, does it use the defense sprite? Does the defense ship even need a sprite?

And since the `defense` is a ship, can the ship only have turrets, or will the object sprite rotate to fire guns?

Reply:
The sprite specified in ship is the sprite which is shown when the ship is alive.
After death the sprite of the object becomes visible following your idea:
> And to add this now, I think that instead of blowing up and disappearing like ships, these platforms should have death sprites that are left behind after the explosion (and the transition from normal sprite to death sprite would be obscured by the final explosion).

> And since the `defense` is a ship, can the ship only have turrets, or will the object sprite rotate to fire guns?

Stations are not completely immobile. If you give them steering then they will be able to turn around. They are just not using thrusters(even if installed).

Reply:
Cool. Thanks for the clarification.
When do defenses "respawn"? Upon landing, or upon the day changing?

Reply:
day change

Reply:
Story reasons, but at that point an event can just handle it, I suppose.